doc/installation-ce.rst

```diff
@@ -16,58 +16,69 @@ The easy way to install the dependencies is to use `pip <https://pypi.python.org
 
 On Debian, you can run ::
 
- aptitude install python-pip python-dev
+ apt update
+ apt install --no-install-recommends -y apt-utils
+ apt install --no-install-recommends -y make wget git gcc libc-dev gunicorn sphinx-common sphinx-doc gnupg2 libsasl2-dev libldap2-dev libssl-dev python3-pip python3-dev
 
 You can then install django and the dependencies ::
 
- pip install -r requirements.txt
+ pip3 install --upgrade pip wheel setuptools
+ pip3 install --upgrade six python-daemon suricatactl
+ pip3 install django-bootstrap3==11.1.0 elasticsearch-curator==5.6
+ pip3 install -r requirements.txt
 
 To use the suri_reloader script which is handling suricata restart, you will also need
 pyinotify ::
 
- pip install pyinotify
-
-It has been reported that on some Debian system forcing a recent GitPython is required ::
-
- pip install gitpython==0.3.1-beta2
+ pip3 install pyinotify
 
 You will also potentially needs the gitdb module ::
 
- pip install gitdb
+ pip3 install gitdb
 
 
 For npm and webpack, you need a stable version of npm and webpack version 3.11. On Debian
 you can do ::
+ 
+Install npm::
+ apt install --no-install-recommends -y npm
+
+Install nodejs dependencies for Scirius:
+::
 
- sudo apt-get install npm
- sudo npm install -g npm@latest webpack@3.11
  npm install
+ mkdir /var/log/scirius/
+ touch /var/log/scirius/elasticsearch.log
+ chown -R user:user /var/log/scirius
+ npm install -g webpack@3.11
+ webpack
```

Comment:
It seems running the webpack at the end is better and it makes sense as all deps will have been installed.

Reply:
done